### ISSUE TITLE
[Microsoft.Azure.WebJobs.Extensions.EventGrid] Internal queuing

### ISSUE THREAD

**Issue by alrod** [Azure/azure-sdk-for-net#17756]
**Library or service name.**
What library or service is this request related to? 
Microsoft.Azure.WebJobs.Extensions.EventGrid
 
**Is your feature request related to a problem? Please describe.**
Without internal queuing, we are going to process all events in parallel
and return 500 if there's at least one failure...which will cause EventGrid to resend the entire payload.

This issue came from the comment in the code:
https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/eventgrid/Microsoft.Azure.WebJobs.Extensions.EventGrid/src/EventGridExtensionConfigProvider.cs#L190

**Comment by JimSuplizio** [comment]
Hi @alrod, we deeply appreciate your input into this project. Regrettably, this issue has remained inactive for over 2 years, leading us to the decision to close it. We've implemented this policy to maintain the relevance of our issue queue and facilitate easier navigation for new contributors. If you still believe this topic requires attention, please feel free to create a new issue, referencing this one. Thank you for your understanding and ongoing support.
